packages/@aws-cdk/aws-eks/test/integ.eks-helm-asset.ts

```diff
@@ -19,7 +18,7 @@ class EksClusterStack extends TestStack {
     });
 
     // just need one nat gateway to simplify the test
-    this.vpc = new ec2.Vpc(this, 'Vpc', { maxAzs: 3, natGateways: 1 });
```

Comment:
why remove `maxAzs`?

Reply:
for environment agnostic stacks `maxAzs` can only be `2`.